src/GafferSceneUI/VisualiserTool.cpp

```diff
@@ -581,6 +581,10 @@ auto stringFromValue = []( auto &&value ) -> std::string
 	{
 		return fmt::format( "{:.3f}, {:.3f}, {:.3f}", value.x, value.y, value.z );
 	}
+	else if constexpr( std::is_same_v<T, Quatf> )
+	{
+		return fmt::format( "( {:.3f}, {:.3f}, {:.3f} ), {:.3f}", value.v.x, value.v.y, value.v.z, value.r );
```

Comment:
> The primitive variable inspector lists the imaginary (x, y, z) part first and then the real part. Imath quatf constructors take the real part first. I settled on matching the primitive inspector order, with the imaginary part in parenthesis to hopefully give a hint that those are the imaginary parts.

We definitely want a consistent presentation for quaternions throughout Gaffer, and I like the usage of the parenthesis to disambiguate. But it is a bit odd to drag the value to the PythonEditor and find it reordered in the `Quatf` constructor there. Is there any merit in changing the PrimitiveInspector and using `w x y z` order in both?

Reply:
Yeah, maybe reordering the columns in the Primitive Inspector is a good way to go. I've done that in https://github.com/GafferHQ/gaffer/pull/6368/commits/dabb2ba64d08b498d6b051c92df47dff288a6340 and changed the order of values for the visualiser tool in https://github.com/GafferHQ/gaffer/pull/6368/commits/4895d6d9046a2737f772d08553186a854891d30c